core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -401,17 +401,20 @@ is_account_enabled(Account) ->
 
     end.
 
--spec is_account_expired(api_binary()) -> boolean().
+-spec is_account_expired(api_binary()) -> 'false' | {'true', integer()}.
```

Comment:
use `gregorian_seconds()` instead to indicate the type of integer this represents.